org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -115,16 +116,18 @@ class Translator {
      * and on the hub translator (this) for verification.
      */
     _subscribe(subscriptionInfo) {
-        var requestPath = '/subscription/' + subscriptionInfo.controlId;
-        return this._makeRequest(subscriptionInfo.endpointUri, requestPath, 'POST', '');
+        var requestPath = this._subcriptionPath + '?deviceId=' + subscriptionInfo.controlId 
+                                            + '&subscriptionURL=' + subscriptionInfo.callbackUrl;
+        return this._makeRequest(subscriptionInfo.endpointUri, requestPath, 'POST', "");
```

Comment:
did you intend to use the passed in uri as the uri to call? why would the end user have this info?